### PR TITLE
Add chaos test coverage for ARM

### PR DESCRIPTION
**Pull request by chaitanyaenr**
This commit adds support to run chaos tests on OCP 4.14 ARM.

### CODE REVIEW

File: ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml [hunk 1]
```diff
@@ -1,4 +1,76 @@
...
+    org: redhat-chaos
+    repo: krkn-hub
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws
+    ci-operator.openshift.io/variant: ARM
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-redhat-chaos-krkn-hub-main-ARM-krkn-hub-tests
+  spec:
```
Comment (paigerube14) [inline ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml:17]: lets be sure to try to add the slack integration here

Comment (chaitanyaenr) [reply ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml:28]: Done! Thanks!

File: ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml [hunk 2]
```diff
@@ -13,17 +85,6 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests
-  reporter_config:
```
Comment (paigerube14) [inline ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml:16]: let's add this back

Comment (chaitanyaenr) [reply ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml:16]: name: periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests is now part of the list, the auto config generation seems to somehow added arm in place of the existing one and added a new config for periodic-ci-redhat-chaos-krkn-hub-main-krkn-hub-tests

File: ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml (hunk 1 above)
Comment (chaitanyaenr) [reply ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml:28]: Missed it, should be in now! Thanks.

**Comment by chaitanyaenr** [comment]
/pj-reharse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests

File: ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__arm.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: krkn-hub-tests
```
Comment (paigerube14) [inline ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__arm.yaml:30]: think we need to rename tis to be different than other test in the main file

File: ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml (hunk 2 above)
Comment (paigerube14) [reply ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml:16]: See my comment in the config file about this that it wont replace but add a new item to the list

File: ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__arm.yaml (hunk 3 above)
Comment (chaitanyaenr) [reply ci-operator/config/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main__arm.yaml]: Ack, done. Thanks.

File: ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml (hunk 2 above)
Comment (paigerube14) [reply ci-operator/jobs/redhat-chaos/krkn-hub/redhat-chaos-krkn-hub-main-periodics.yaml:16]: Think you'll need to regenerate these files using 'make jobs'

**Comment by chaitanyaenr** [comment]
/pj-reharse periodic-ci-redhat-chaos-krkn-hub-main-arm-krkn-hub-tests

**Pull request closed without merging**